src/test/java/teammates/test/cases/action/InstructorCourseStudentDetailsEditSaveActionTest.java

```diff
@@ -70,7 +71,9 @@ public void testExecuteAndPostProcess() throws Exception {
                      r.getDestinationWithParams());
         
         assertFalse(r.isError);
-        assertEquals(Const.StatusMessages.STUDENT_EDITED, r.getStatusMessage());
+        assertEquals(Const.StatusMessages.STUDENT_EDITED_AND_EMAIL_SENT, r.getStatusMessage());
+        
+        verifyNumberOfEmailsSent(a, 1);
```

Comment:
Checking just the email amount is insufficient. Check also that the subject and the recipient is correct.